src/docfx/config/Config.cs

```diff
@@ -49,12 +49,12 @@ internal class Config
         /// <summary>
         /// Just for backward compatibility, the output site path prefix
         /// </summary>
-        public readonly string SiteBasePath = string.Empty;
+        public readonly string SiteBasePath = ".";
 
         /// <summary>
         /// Just for backward compatibility, the source path prefix
         /// </summary>
-        public readonly string SourceBasePath = string.Empty;
+        public readonly string SourceBasePath = ".";
```

Comment:
Would you please check the code if there are any places using string.IsNullorEmpty() for these two field?